<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VPlayerSlider</class>
 <widget class="QWidget" name="VPlayerSlider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VPlayerSlider</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_currenttime, #label_totaltime
{
	background-color: rgba(255, 255, 255, 20%);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_play">
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_currenttime">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider_player">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_totaltime">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
